lib/modules/dosomething/dosomething_payment/dosomething_payment.module

```diff
@@ -210,6 +210,10 @@ function dosomething_payment_form_submit($form, &$form_state) {
   $values['amount'] = $values['amount'] * 100;
   // Send Payment request to Stripe API.
   if (dosomething_payment_post_payment_stripe($values)) {
+    // Send stathat info.
+    if (module_exists('stathat')) {
+      stathat_send_ez_count('drupal - Donate', 1);
```

Comment:
Minor, but I'm not sure if the module handles when the ez key hasn't been entered. Is the plan to manually enter the key on stage and prod?